src/server/scripts/Spells/spell_hunter.cpp

```diff
@@ -1011,6 +1013,41 @@ class spell_hun_tnt : public SpellScriptLoader
             return new spell_hun_tnt_AuraScript();
         }
 };
+// -186257 - Aspect of the cheetah
+class spell_hun_aspect_cheetah : public SpellScriptLoader
+{
+public:
+    spell_hun_aspect_cheetah() : SpellScriptLoader("spell_hun_aspect_cheetah") { }
+
+    class spell_hun_aspect_cheetah_AuraScript : public AuraScript
+    {
+
+        PrepareAuraScript(spell_hun_aspect_cheetah_AuraScript);
+
+        bool Validate(SpellInfo const* /*spellInfo*/) override
+        {
+            if (!sSpellMgr->GetSpellInfo(SPELL_HUNTER_ASPECT_CHEETAH_FAST))
```

Comment:
Exactly ! Need only validate spells which are called in spell/aurascript
Also you need remove one of double empty lines at 1040 before `Register` and it's perfect, good job :)